test/display_test.py

```diff
@@ -65,20 +65,53 @@ def todo_test_flip(self):
 
         self.fail()
 
-    def todo_test_get_active(self):
+    def test_get_active(self):
+        """Test the get_active function"""
+
+        #Initially, the display is not active
+        pygame.quit()
+        pygame.display.quit()
+        self.assertEqual(pygame.display.get_active(), False)
+
+        #get_active defaults to true after a set_mode
+        pygame.init()
```

Comment:
likewise should change this to `pygame.display.init()` since we are just testing the display module.